exercises/sum.c

```diff
@@ -0,0 +1,20 @@
+/*
+  Write a program that takes as input two numbers and print the sum.
+
+  Output:
+  Insert the first number: 1
+  Insert the second number: 2
+  Sum: 3
+*/
+#include <stdio.h>
+
+int main(){
+  //dec
+  int a=0,b=0;
+  printf("insert one number\n");
+  scanf("%d",a);
```

Comment:
il codice che legge le variabili potrebbe essere messo in una funzione a parte, visto che è identico in entrambi i casi